src/theme/components/link.ts

```diff
@@ -0,0 +1,45 @@
+import { ComponentMultiStyleConfig, SystemStyleObject } from '@chakra-ui/react';
+
+import { fontSizes, lineHeights, space } from '../shared';
+
+const linkBaseStyle: SystemStyleObject = {
+  lineHeight: lineHeights.display,
+  fontSize: fontSizes.base,
```

Comment:
Would you mind doble checking with the designer this new text variation?

We must use the tokens that we already have. They need to be in sync with the tokens instead creating new ones. Please doble check with Enrique or Sergiu and use the typography tokens that we have. It's important not to start creating new rules and new variations.

Thank you 🙏🏽

Reply:
There's no new token was created by this PR. If you meant the combination of `lineHeight` and `fontSize` should be included in the `typography` tokens, then it's a different story

Reply:
I already opened a discussion where the idea is to use only the `typography tokens`. Therefore, I need to sync with the designer and adjust those tokens. But nothing that can block you. You can go ahead with this and after we set the tokens we could open a a PR with some improvements.

Thank you!